tests/cli/test_compose_live-iso.sh

```diff
@@ -15,40 +15,66 @@ set -e
 CLI="${CLI:-./src/bin/composer-cli}"
 
 rlJournalStart
-    rlPhaseStartSetup
-        rlAssertExists $QEMU_BIN
-    rlPhaseEnd
-
     rlPhaseStartTest "compose start"
         rlAssertEquals "SELinux operates in enforcing mode" "$(getenforce)" "Enforcing"
 
+        TMP_DIR=$(mktemp -d /tmp/composer.XXXXX)
+        SSH_KEY_DIR=$(mktemp -d /tmp/composer-ssh-keys.XXXXXX)
+
+        rlRun -t -c "ssh-keygen -t rsa -N '' -f $SSH_KEY_DIR/id_rsa"
+        PUB_KEY=$(cat "$SSH_KEY_DIR/id_rsa.pub")
+
+        cat > "$TMP_DIR/with-ssh.toml" << __EOF__
+name = "with-ssh"
+description = "HTTP image with SSH"
+version = "0.0.1"
+
+[[packages]]
+name = "httpd"
+version = "*"
+
+[[packages]]
+name = "openssh-server"
+version = "*"
+
+[[packages]]
+name = "beakerlib"
+version = "*"
+
+[customizations.services]
+enabled = ["sshd"]
+
+[[customizations.user]]
+name = "root"
+key = "$PUB_KEY"
+
+[customizations.kernel]
+append = "custom_cmdline_arg"
+__EOF__
+
+        rlRun -t -c "$CLI blueprints push $TMP_DIR/with-ssh.toml"
+
         # NOTE: live-iso.ks explicitly disables sshd but test_cli.sh enables it
-        UUID=`$CLI compose start example-http-server live-iso`
+        UUID=$($CLI compose start with-ssh live-iso)
         rlAssertEquals "exit code should be zero" $? 0
 
-        UUID=`echo $UUID | cut -f 2 -d' '`
+        UUID=$(echo "$UUID" | cut -f 2 -d' ')
     rlPhaseEnd
 
     rlPhaseStartTest "compose finished"
-        wait_for_compose $UUID
-
-        rlRun -t -c "$CLI compose image $UUID"
-        IMAGE="$UUID-live.iso"
-    rlPhaseEnd
-
-    rlPhaseStartTest "Start VM instance"
-        boot_image "-boot d -cdrom $IMAGE" 120
```

Comment:
this breaks the existing behavior of being able to execute this script on a bare metal system, build the image, boot a VM from it and ssh into the test VM to verify a few settings. Preferably this would be disabled with an if statement when running inside Cockpit CI. If you want you can pass additional variables from the Python caller.